.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+CHANGELOG: `no-impact` / `Added` / `Changed` / `Deprecated` / `Removed` / `Fixed` / `Security`
+
+Describe how these changes affect the customer in one line. This is the thing you are going to add to a changelog.txt file.
+
+> CHANGELOG entries will be pulled directly into the release changelog, so use
+customer-facing language. Start with one of the active verbs above,
+write a full sentence, and end with a period. If these changes do not
+impact the customer, then this line should read CHANGELOG: no-impact.
+
+### Description
+Describe why you're making this change, in plain English.
+
+### Testing & Repro steps
+* In the case of bugs, describe how to replicate
+* If any manual tests were done, document the steps and the conditions to replicate
+* Call out any important/ relevant unit tests, e2e tests or integration tests you have added or are adding
+
+### Links
+Include any links here that might be helpful for people reviewing your PR (GH issues, API docs, external benchmarks, tools docs, etc). If there are none, feel free to delete this section.
+
+### PR Checklist
+
+Please fill in this checklist PRIOR to opening a PR.
+
+* [ ] lint
+* [ ] testing
```

Comment:
These two will fail at CI so I don't think we need to call them out

Reply:
hm, I've seen PRs before that don't add these. My own PRs sometimes don't lint 🙈 bc I forget to do it; and I think it's fine to use the CI as a linter (💸 concern maybe?).

---

On the testing front, I have also seen changes go in without a lot of test coverages (certainly the odd PR, not the norm) but I want testing/ quality to be forefront of my mind 

---

I think all the `PR Checklist` items can be re-worded to better describe that ^ 😆

Reply:
I think lint doesn't belong here, because it's automated and there shouldn't need to be any user action to check off.
As far as testing goes; maybe something like 'any needed test coverage for affected area'

Reply:
removed per feedback